packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.scss

```diff
@@ -221,8 +221,19 @@ $DropDown-item-height: 32px;
   color: $ms-color-black;
 
   @include highContrastListItemState;
+
+  &.itemIsDisabled {
+    color: $ms-color-neutralTertiary;
+  }
 }
 
+.dropdownCheckbox {
+  &.item.itemIsSelected.itemIsDisabled {
+    background-color: $ms-color-white;
+  }
+}
```

Comment:
Want to draw attention to these lines in particular since I haven't added much to styles before. Want to make sure I did this right

Reply:
Ok cool. I'll take a closer look tomorrow.

Reply:
Two things:
1. I think background color should be transparent to match the default bg color. It will pick the white up from the callout bg color.
2. You can stick this bg style up under your other &.itemIsDisabled on line 226.

Reply:
Good call on transparent. Regarding # 2 -- want to confirm that a disabled selected item in a single-select dropdown should have transparent bg as well. Since the single-select dropdown doesn't have checkboxes, it's not obvious (within the callout) that the disabled option is selected.

Transparent: 
![checkbox drodown disabled transparent](https://user-images.githubusercontent.com/15041132/34308829-f1ed7e64-e703-11e7-9732-b72de2571563.PNG)

Gray:
![checkbox drodown disabled](https://user-images.githubusercontent.com/15041132/34308830-f1fd6c70-e703-11e7-9cd2-12e8a768319a.PNG)

Reply:
Hm good question. Not obvious it's selected, but also I'm not sure that there is a selection state for disabled option in a single-select Dropdown. There isn't one in the toolkit that I can see. @betrue-final-final , thoughts?

Reply:
I think there's no selection state because the user couldn't re-select the disabled option once they changed it. So it really shouldn't be selected in the first place.

Reply:
@erichdev I confirmed with Ben - there is no selection state for a disabled option. If you go ahead and make the changes I suggested, we can merge this PR in.